sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/SamplesConstants.cs

```diff
@@ -96,7 +96,7 @@ public static class SamplesConstants
         /// <summary>
         /// Path for component.
```

Comment:
nit: update comment to "Name of component"

Reply:
Good catch!